test/server/cache-module.js

```diff
@@ -45,7 +45,7 @@ describe('cacheModule Tests', function () {
       done();
     });
   });
-  it('Setting several keys then calling .flushAll() should remove all keys', function (done) {
```

Comment:
What happened to the `flushAll` test? Did `flushAll` go away? Oh, was the test incorrectly named? That makes sense. Thanks for fixing that.

Reply:
I'm not sure - I believe it was incorrectly named as I couldn't find any references to it!